method.js

```diff
@@ -0,0 +1,32 @@
+import { bookdiv } from './modules/getelement.js';
+
+export default class Book {
+  constructor() {
+    this.Books = [];
+  }
+
+  /// display the books in html page
+  display() {
+    bookdiv.innerHTML = '';
+    for (let i = 0; i < this.Books.length; i += 1) {
+      const listofbook = document.createElement('div');
+      listofbook.classList.add('book');
+      if (i % 2 === 0) {
+        listofbook.classList.add('bg-danger');
+      } else {
+        listofbook.classList.add('bg-light');
+      }
+      listofbook.innerHTML = `<p>"${this.Books[i].book}" by
+            ${this.Books[i].author}</p>
+            <button id=${i} class="rem-btn" >Remove</button><br> <br>`;
+      bookdiv.appendChild(listofbook);
+    }
+  }
+
+  // remove the books
+  removebook(id) {
+    this.Books.splice(id, 1);
+    window.localStorage.setItem('books', JSON.stringify(this.Books));
+    this.display();
+  }
+}
```

Comment:
- [ ]  Good job your application is working, we can add and remove items :+1:  However there is something you need to fix :cry: When we refresh the browser your list of added books disappears and is not displaying anymore until we add another book. Please fix it in order to make your app work as expected :+1: 

![Screenshot from 2022-08-08 23-43-42](https://user-images.githubusercontent.com/6615394/183533070-993b86e3-473a-49c9-b716-5aa15c3720f4.png)
![Screenshot from 2022-08-08 23-43-17](https://user-images.githubusercontent.com/6615394/183533076-5257a8fd-f7d2-4923-95ef-ec88b66cc759.png)


- [ ] The project requirement mentions that all other js files except `index.js`, `index.html`, and `index.css`  should be in **modules folder** Please move your method files to the modules folder :+1: